.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,3 +12,10 @@
 - 
 - 
 - 
+
+## Pre-requisites
+
+Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
```

Comment:
Is it possible to use a relative link here, perhaps?

Reply:
yes, done